Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -873,9 +963,285 @@ struct HFTrackIndexSkimsCreator {
+        MY_DEBUG_MSG(isProtonFromLc, LOG(INFO) << "proton " << protonLabel << ": rejected due to minNUmberOfCrossedRows " << bach.tpcNClsCrossedRows() << " (cut " << i_minCrossedRows << ")");
+        continue;
+      }
+      MY_DEBUG_MSG(isProtonFromLc, LOG(INFO) << "KEPT! proton from Lc with daughters " << protonLabel);
+
+      auto bachTrack = getTrackParCov(bach);
+      // now we loop over the V0s
+      for (const auto& v0 : V0s) {
+        MY_DEBUG_MSG(1, LOG(INFO) << "*** Checking next K0S");
+        // selections on the V0 daughters
+        const auto& posTrack = v0.posTrack_as<MyTracks>();
+        const auto& negTrack = v0.negTrack_as<MyTracks>();
+#ifdef MY_DEBUG
+        auto labelPos = posTrack.mcParticleId();
+        auto labelNeg = negTrack.mcParticleId();
+        bool isK0SfromLc = isK0SfromLcFunc(labelPos, labelNeg, v_labelK0Spos, v_labelK0Sneg);
+
+        bool isLc = isLcK0SpFunc(protonLabel, labelPos, labelNeg, v_labelProton, v_labelK0Spos, v_labelK0Sneg);
+#endif
+        MY_DEBUG_MSG(isK0SfromLc, LOG(INFO) << "K0S from Lc found, posTrack --> " << labelPos << ", negTrack --> " << labelNeg);
+
+        MY_DEBUG_MSG(isK0SfromLc && isProtonFromLc,
+                     LOG(INFO) << "ACCEPTED!!!";
+                     LOG(INFO) << "proton belonging to a Lc found: label --> " << protonLabel;
+                     LOG(INFO) << "K0S belonging to a Lc found: posTrack --> " << labelPos << ", negTrack --> " << labelNeg);
+
+        MY_DEBUG_MSG(isLc, LOG(INFO) << "Combination of K0S and p which correspond to a Lc found!");
+
+        if (b_TPCRefit) {
+          if (!(posTrack.trackType() & o2::aod::track::TPCrefit) ||
+              !(negTrack.trackType() & o2::aod::track::TPCrefit)) {
+            MY_DEBUG_MSG(isK0SfromLc, LOG(INFO) << "K0S with daughters " << labelPos << " and " << labelNeg << ": rejected due to TPCrefit");
+            continue;
+          }
+        }
+        if (posTrack.tpcNClsCrossedRows() < i_minCrossedRows ||
+            negTrack.tpcNClsCrossedRows() < i_minCrossedRows) {
+          MY_DEBUG_MSG(isK0SfromLc, LOG(INFO) << "K0S with daughters " << labelPos << " and " << labelNeg << ": rejected due to minCrossedRows");
+          continue;
+        }
+        //
+        // if (posTrack.dcaXY() < d_dcaXYPosToPV ||   // to the filters?
+        //     negTrack.dcaXY() < d_dcaXYNegToPV) {
+        //   continue;
+        // }
+        //
+        if (posTrack.pt() < d_ptMin || // to the filters? I can't for now, it is not in the tables
+            negTrack.pt() < d_ptMin) {
+          MY_DEBUG_MSG(isK0SfromLc, LOG(INFO) << "K0S with daughters " << labelPos << " and " << labelNeg << ": rejected due to minPt --> pos " << posTrack.pt() << ", neg " << negTrack.pt() << " (cut " << d_ptMin << ")");
+          continue;
+        }
+        if (abs(posTrack.eta()) > d_etaMax || // to the filters? I can't for now, it is not in the tables
+            abs(negTrack.eta()) > d_etaMax) {
```

Comment:
`std::abs`

Reply:
Everywhere in this class only "abs" is used.